Meta/run.sh

```diff
@@ -47,19 +61,30 @@ $SERENITY_EXTRA_QEMU_ARGS
 
 export SDL_VIDEO_X11_DGAMOUSE=0
 
+[ ! "${SERENITY_BUILD+is_set}" ] && {
+    if [ -d "$SERENITY_ROOT/Build" ]; then
+        SERENITY_BUILD="$SERENITY_ROOT/Build"
+    elif [ -d "$SERENITY_ROOT/build" ]; then
+        SERENITY_BUILD="$SERENITY_ROOT/build"
+    fi
+}
+: "${SERENITY_BUILD:='.'}"
```

Comment:
Can't you put this inside the previous block? Something like a final `else` if neither directory exists.

Reply:
Why? If `SERENITY_BUILD` is not set, `SERENITY_ROOT/Build` does not exist/is not a directory and `SERENITY_ROOT/build` does not exist/is not a directory, then `: "${SERENITY_BUILD:='.'}" will set it to `.` (intended behaviour: read commit message).

We could fail in that case, I guess. Is that what you mean?

Reply:
> Can't you put this inside the previous block?

Oh, I missed this part: no, I can't: if the variable is set, but empty, I still want to set `SERENITY_BUILD` to `.`